ext/opentelemetry-ext-grpc/src/opentelemetry/ext/grpc/__init__.py

```diff
@@ -12,13 +12,61 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 # pylint:disable=import-outside-toplevel
 # pylint:disable=import-self
+from concurrent import futures
 # pylint:disable=no-name-in-module
 # pylint:disable=relative-beyond-top-level
-
+from contextlib import contextmanager
+from wrapt import wrap_function_wrapper as _wrap
 from opentelemetry import trace
+from opentelemetry.auto_instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.ext.grpc.grpcext import intercept_channel, intercept_server
 from opentelemetry.ext.grpc.version import __version__
+from opentelemetry.instrumentation.utils import unwrap
+
+
+
+class GrpcInstrumentorServer(BaseInstrumentor):
+    def __init__(self):
+        self.server = None
+
+    def _instrument(self, **kwargs):
+        _wrap("grpc", "server", self.wrapper_fn)
+
+    def _uninstrument(self, **kwargs):
+        unwrap("grpc", "server")
+
+    def wrapper_fn(self, original_func, instance, args, kwargs):
+        self.server = original_func(*args, **kwargs)
+        self.server = intercept_server(self.server, server_interceptor())
```

Comment:
I assume `grpc.server()` returns a server, so the wrapper function should do the same (and setting `self.server` should be unnecessary). So something like 
```suggestion
        server = original_func(*args, **kwargs)
        return intercept_server(server, server_interceptor())
```

Reply:
I noticed in some of the docs a reference to what the intercept_server method return was needed. So I included it for that reason.